tracing.go

```diff
@@ -83,5 +83,12 @@ func trace(parent context.Context, name string) (opentracing.Span, context.Conte
 
 	span.SetTag("source", "shim")
 
+	// This is slightly confusing: when tracing is disabled, trace spans
+	// are still created - but the tracer used is a NOP. Therefore, only
+	// display the message when tracing is really enabled.
+	if tracing {
+		shimLog.Debugf("created span %v", span)
```

Comment:
`WithField` :question:  :running_man:

Reply:
Good question. Yes, that would make sense but it would also require changes to the agent and the [tracing test](https://github.com/kata-containers/tests/blob/master/tracing/tracing-test.sh#L107..L115) which is currently hard-coded to look for `created span $span`. If we make this call structured, we'd also have to deal with extracting the span field which might be in any structured log field/position (well potentially).

Reply:
that make sense, thanks